app/cdap/components/shared/PlusButtonModal/PlusButtonModal.scss

```diff
@@ -105,19 +105,19 @@ $horizontal-tab-height: 45px;
             n column layout : (n-1 * 10px);
         */
 
-        @media (min-width: 1601px) {
+        @media (min-width: 2601px) {
```

Comment:
Since I added a new button to the pipeline card in resource center, I'm adjusting these values so that 3 buttons can show on the same line